### PR TITLE
[Git] Add new v2.25.0 pretty-formats

### PR DESCRIPTION
**Pull request by michaelblyons**
See
- git/git@c22f63c
- git/git@d37cfe3

### CODE REVIEW

**Comment by jrappen** [comment]
@michaelblyons maybe update the test for gitconfig?

**Comment by michaelblyons** [comment]
Okay. I found a different bug while making the tests. Fixed that, too.

File: Git Formats/Git Common.sublime-syntax [hunk 1]
```diff
@@ -283,6 +283,7 @@ contexts:
     - include: pretty-formats-wrap-formatting
     - include: pretty-formats-string-formatting
     - include: pretty-formats-trailers
+    - match: \S
```
Comment (deathaxe) [inline Git Formats/Git Common.sublime-syntax]: What is that `-match: \S` for? We stay in that context until the `(?!\S)` on line 238 is matched. Removing it doesn't make a difference in your test cases.

Comment (michaelblyons) [reply Git Formats/Git Common.sublime-syntax]: Good catch! It was added intentionally, along with the Line 394 test. But the test must only have been broken while I was tinkering with it. I'll take that out.

File: Git Formats/syntax_test_git_config [hunk 2]
```diff
@@ -377,14 +381,26 @@ stray-bracket]
...
+#                             ^^ constant.other.placeholder.pretty-formats.git
+    foo = log --pretty=tformat:%h
+#                      ^^^^^^^ keyword.declaration.pretty-formats.git
+#                             ^ keyword.operator.assignment.option.pretty-formats.git
+#                              ^^ constant.other.placeholder.pretty-formats.git
     foo = log --pretty=%a
 #                      ^^ - constant
-    foo = log --pretty=%aD
+    foo = log --pretty=%aDx%cD
 #                      ^^^ constant.other.placeholder.pretty-formats.git
```
Comment (jrappen) [inline Git Formats/syntax_test_git_config]: @michaelblyons I assume you should change this as well, when changing the previous line.

**Comment by wbond** [comment]
Thanks!

**Pull request merged**